apps/bluetooth-music/README.md

```diff
@@ -0,0 +1,49 @@
+# Bluetooth music player
+
+This is the app for play bluetooth music.
+
+## App URL
+
+* scheme and host:
+
+  * `yoda-app://bluetooth-music`
+
+* paths and queries:
+
+  * `start`: Start/resume play.
+
+  * `pause`: Pause music.
+
+  * `stop`: Stop music.
+
+  * `next`: Play next song.
+
+  * `prev`: Play previous song.
+
+  * `like`: Add current song to favorite.
+
+  * `info`: Query current song's information.
+
+|scheme and host|path|query's key|query's value|remark|
+|---|---|---|---|---|
+||start|||Start/resume play.|
+||pause|||Pause music.|
+||stop|||Stop music.|
+|yoda-app://bluetooth-music|next|||Play next song.|
+||prev|||Play previous song.|
+||like|||Add current song to favorite.|
+||info|||Query current song's information.|
+
+## YodaOS event
+
+* status event
+
+  * flora subscribe name: `yodaos.apps.bluetooth.playback-status`
+
+  * flora message format: ```String: PLAYING|PAUSED|STOPPED```
+
+* music info event
+
+  * flora subscribe name: `yodaos.apps.bluetooth.music-info`
+
+  * flora message format: ```Object { title: xxx, artist: yyy, album: zzz }```
```

Comment:
I don't think caps support key value pairs. Do you mean JSON string?

Reply:
JSON_OBJECT